### PR TITLE
Christoph/addfos

### PR DESCRIPTION
**Pull request by chrished**
added mag field of study to database. There is one minor issue @f-hafner: when I close the database connection I get an error that the connection is already closed. I suspect you know why? 

Let me know if I should change anything else

### CODE REVIEW

File: src/dataprep/main/load_proquest/magfos_to_db.py [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+                                    , score NUMERIC
+                                """
+                    )
+
+    # Make index and clean up
+    con.execute("CREATE UNIQUE INDEX idx_pq_magfos ON pq_magfos (goid ASC,fieldrank ASC, FieldOfStudyId ASC)")
+
+    analyze_db(con)
+
+    con.close()
```
Comment (f-hafner) [inline src/dataprep/main/load_proquest/magfos_to_db.py:73]: Maybe you have to put the `con.close()` out of the context manager? at least this is how I usually do it.

File: src/dataprep/main/load_proquest/magfos_to_db.py [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+Read in mag field of study of dissertation abstracts (estimated on TDM studio) and save to database.
+
+Create table in database:
+- pq_magfos with the estimated fields of study
+
+pq_magfos data schema is:
+
+goid, fieldrank INTEGER, FieldOfStudyId INTEGER, score NUMERIC 
+
+unique index on goid and FieldOfStudyId (multiple fields per goid)
```
Comment (f-hafner) [inline src/dataprep/main/load_proquest/magfos_to_db.py:15]: can you quickly describe what is fieldrank?

Comment (f-hafner) [inline src/dataprep/main/load_proquest/magfos_to_db.py:15]: statement in line 15 contradicts line 69?

**Review by f-hafner — CHANGES_REQUESTED** [review]

File: src/dataprep/main/load_proquest/magfos_to_db.py (hunk 1 above)
Comment (chrished) [reply src/dataprep/main/load_proquest/magfos_to_db.py]: see  last commit

File: src/dataprep/main/load_proquest/magfos_to_db.py (hunk 2 above)
Comment (chrished) [reply src/dataprep/main/load_proquest/magfos_to_db.py:18]: done

Comment (chrished) [reply src/dataprep/main/load_proquest/magfos_to_db.py:18]: corrected

**Review by f-hafner — APPROVED** [review]

**Pull request merged**